auditlog/models/rule.py

```diff
@@ -378,6 +406,12 @@ def write_full(self, vals, **kwargs):
                 .with_context(prefetch_fields=False)
                 .read(fields_list)
             }
+            if self.env.user in users_to_exclude_ids:
+                return result
+            for d in vals.keys():
+                if d not in fields_to_exclude_ids:
```

Comment:
```suggestion
            if self.env.user in users_to_exclude:
                return result
            for d in vals.keys():
                if d not in fields_to_exclude:
```
Same remark regarding the fields to exclude: why not filtering out these fields from the log?